ang/civicase/BulkActionsMessage.html

```diff
@@ -0,0 +1,5 @@
+<div class="civicase__bulkactions-message" ng-show="selectedItems > 0 && showCheckboxes">
+  <div class="alert">
+    You have selected <strong>{{selectedItems}}</strong> <ng-pluralize count="selectedItems" when="{'one': 'item', 'other': 'items'}"></ng-pluralize>. <a ng-show="totalCount !== selectedItems" href="#" ng-class="{'civicase__link-disabled': !isSelectAllAvailable}" ng-click="select('all'); $event.stopPropagation(); $event.preventDefault();">Select all {{totalCount}} that match the search</a> <a ng-show="totalCount === selectedItems" href="#"  ng-class="{'civicase__link-disabled': !isSelectAllAvailable}" ng-click="select('none'); $event.stopPropagation(); $event.preventDefault();">Clear All</a>
```

Comment:
`$event.stopPropagation(); $event.preventDefault();` can be moved inside `select`.